### PR TITLE
Set cmv_id in port_drayage_plugin to vehicle_id parameter

### PR DESCRIPTION
**Pull request by JonSmet**
# PR Details
## Description

This PR update port_drayage_plugin to set its cmv_id using the 'vehicle_id' global parameter, which includes the vehicle's license plate information. As part of this update, cmv_id within port_drayage_plugin has been changed from an unsigned long to a string. Similarly, V2XHub expects cmv_id to be a string value.

## How Has This Been Tested?

Built and tested locally with development config; also unit tested.

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: port_drayage_plugin/src/port_drayage_plugin.cpp [hunk 1]
```diff
@@ -38,12 +38,8 @@ namespace port_drayage_plugin
...
         bool enable_port_drayage;
         _pnh->param<bool>("enable_port_drayage", enable_port_drayage, false);
-
-        // Read in 'cmv_id' parameter as a string, then convert to an unsigned long before initializing the PortDrayageWorker object
-        std::string cmv_id_string;
-        _pnh->param<std::string>("cmv_id", cmv_id_string, "0");
-        unsigned long cmv_id = std::stoul(cmv_id_string);
-
+        std::string cmv_id;
+        _pnh->getParam("vehicle_id", cmv_id);
```
Comment (msmcconnell) [inline port_drayage_plugin/src/port_drayage_plugin.cpp:42]: This needs to be "/vehicle_id" in order to be loaded from global scope.

File: port_drayage_plugin/config/params.yml [hunk 2]
```diff
@@ -1,5 +1,4 @@
 stop_speed_epsilon: 1.0
-cmv_id: "123" # Test value; CARMA Streets expects cmv_id to be an unsigned long (32-bit unsigned integer)
```
Comment (msmcconnell) [inline port_drayage_plugin/config/params.yml:2]: CARMA Streets expects a 32bit unsigned int. That limits the id string to at most 4 characters. All of our vehicle Ids are longer than 4 characters. Either some use case specific encoding needs to be happening or the message updated to support a larger value.

**Review by msmcconnell — CHANGES_REQUESTED** [review]

File: port_drayage_plugin/src/port_drayage_plugin.cpp (hunk 1 above)
Comment (JonSmet) [reply port_drayage_plugin/src/port_drayage_plugin.cpp]: Fixed

File: port_drayage_plugin/config/params.yml (hunk 2 above)
Comment (JonSmet) [reply port_drayage_plugin/config/params.yml:2]: This comment was from an earlier design decision that required cmv_id to be set to a 32-bit unsigned integer. Later in testing, it was detemrined that the cmv_id should match the vehicle's vehicle_id parameter. V2XHub/CARMA Streets has already been updated to expect and treat the cmv_id value as a string in received MobilityOperation messages for Port Drayage.

**Review by msmcconnell — APPROVED** [review]
LGTM

**Pull request merged**